packages/cactus-cmd-api-server/src/test/typescript/integration/jwt-endpoint-authorization.test.ts

```diff
@@ -74,39 +74,50 @@ test(testCase, async (t: Test) => {
-      "failed to start API server with dynamic plugin imports configured for it...",
-    );
-    t.ok(startResponse, "startResponse truthy OK");
+  test(testCase, async () => {
+    try {
+      expect(expressJwtOptions).toBeTruthy();
+
+      const jwtPayload = { name: "Peter", location: "London" };
+      const tokenGood = await new SignJWT(jwtPayload)
+        .setProtectedHeader({
+          alg: "RS256",
+        })
+        .setIssuer(expressJwtOptions.issuer)
+        .setAudience(expressJwtOptions.audience)
+        .sign(jwtKeyPair.privateKey);
+      // const tokenBad = JWT.sign(jwtPayload, jwtKeyPair);
+
+      const startResponse = apiServer.start();
+      await expect(startResponse).not.toReject;
+      expect(startResponse).toBeTruthy();
 
-    const addressInfoApi = (await startResponse).addressInfoApi;
-    const protocol = apiSrvOpts.apiTlsEnabled ? "https" : "http";
-    const { address, port } = addressInfoApi;
-    const apiHost = `${protocol}://${address}:${port}`;
+      const addressInfoApi = (await startResponse).addressInfoApi;
+      const protocol = apiSrvOpts.apiTlsEnabled ? "https" : "http";
+      const { address, port } = addressInfoApi;
+      const apiHost = `${protocol}://${address}:${port}`;
 
-    const baseOptions = { headers: { Authorization: `Bearer ${tokenGood}` } };
-    const conf = new Configuration({ basePath: apiHost, baseOptions });
-    const apiClient = new ApiServerApi(conf);
-    const resHc = await apiClient.getHealthCheckV1();
-    t.ok(resHc, "healthcheck response truthy OK");
-    t.equal(resHc.status, 200, "healthcheck response status === 200 OK");
-    t.equal(typeof resHc.data, "object", "typeof resHc.data is 'object' OK");
-    t.ok(resHc.data.createdAt, "resHc.data.createdAt truthy OK");
-    t.ok(resHc.data.memoryUsage, "resHc.data.memoryUsage truthy OK");
-    t.ok(resHc.data.memoryUsage.rss, "resHc.data.memoryUsage.rss truthy OK");
-    t.ok(resHc.data.success, "resHc.data.success truthy OK");
-    t.true(isHealthcheckResponse(resHc.data), "isHealthcheckResponse OK");
-    t.end();
-  } catch (ex) {
-    log.error(ex);
-    t.fail("Exception thrown during test execution, see above for details!");
-    throw ex;
-  }
+      const baseOptions = { headers: { Authorization: `Bearer ${tokenGood}` } };
+      const conf = new Configuration({ basePath: apiHost, baseOptions });
+      const apiClient = new ApiServerApi(conf);
+      const resHc = await apiClient.getHealthCheckV1();
+      expect(resHc).toBeTruthy();
+      expect(resHc.status).toEqual(200);
+      expect(typeof resHc.data).toBeTruthy();
+      expect(resHc.data.createdAt).toBeTruthy();
+      expect(resHc.data.memoryUsage).toBeTruthy();
+      expect(resHc.data.memoryUsage.rss).toBeTruthy();
+      expect(resHc.data.success).toBeTruthy();
+      expect(isHealthcheckResponse(resHc.data)).toBe(true);
+    } catch (ex) {
+      log.error(ex);
+      fail("Exception thrown during test execution, see above for details!");
```

Comment:
@awadhana Any reason to remove the re-throw of the exception here? I'm not sure if it makes a difference either way, but wanted to clarify just in case.

Reply:
@petermetz, added it back in :)

Reply:
@awadhana Thank you!